client/components/Balance/BalanceRow.js

```diff
@@ -0,0 +1,19 @@
+import React, { PropTypes } from 'react';
+
+import './style.scss';
+
+const Balance = ({ children }) => {
+  return (
+    <div className="balance-row-container">
```

Comment:
Need to for styling. balance-row sizes itself to its content width. balance-row-container centers it. If we could get away with a gray box that's the full width, we could get rid of a div but tat's what the wireframe shows.